intake_xarray/tests/test_image.py

```diff
@@ -184,3 +184,10 @@ def test_read_images_and_exif_as_glob_with_coerce():
     ds = source.read()
     assert ds['raster'].shape == (3, 256, 256, 3)
     assert ds['EXIF Image ImageWidth'].shape == (3,)
+
+
+def test_read_images_and_persist():
+    pytest.importorskip('skimage')
+    urlpath = os.path.join(here, 'data', 'images', '*')
+    source = ImageSource(urlpath=urlpath, coerce_shape=(256, 256))
+    source.export('./exported')
```

Comment:
better use a temporary directory, else these will pollute the local repo. Shouldn't we also load them again, to check that the export worked?

Reply:
using `tempfile` now

Reply:
how to load again? just with `xr.open_dataset(engine="zarr")` or with `intake`?